### ISSUE TITLE
Postgres - fail to create table with smallserial

### ISSUE THREAD

**Issue by absta** [ariga/atlas#1520]
When applying the SQL
```
create table "ItemConditions" (
    "Id"              smallserial primary key,
    "ConditionValue"  varchar(25)
);
```
The following error is returned
```
-- Planned Changes:
-- Create "ItemConditions" table
CREATE TABLE "ItemConditions" ("Id" smallint NOT NULL DEFAULT nextval('"ItemConditions_Id_seq"'::regclass), "ConditionValue" character varying(25) NULL, PRIMARY KEY ("Id"));
✔ Apply
Error: create "ItemConditions" table: pq: relation "ItemConditions_Id_seq" does not exist
```
According to the [docs](https://atlasgo.io/atlas-schema/sql-types#serial) this type is should be supported?

Please help!

**Comment by a8m** [comment]
Thanks for reporting this, @absta. Will check this now

**Comment by a8m** [comment]
Fixed with #1521, and a new version will be released soon.
